modules/ducktests/tests/ignitetest/services/utils/ssl/ssl_context.py

```diff
@@ -19,20 +19,24 @@
 import os
 
 DEFAULT_PASSWORD = "123456"
-DEFAULT_SERVER_KEYSTORE = "server.jks"
-DEFAULT_CLIENT_KEYSTORE = "client.jks"
-DEFAULT_ADMIN_KEYSTORE = "admin.jks"
 DEFAULT_TRUSTSTORE = "truststore.jks"
+DEFAULT_ROOT = "/opt/"
 
+default_keystore = {
+    'server': 'server.jks',
+    'client': 'client.jks',
```

Comment:
is not used

Reply:
Used in IgniteService  and IgniteApplicationService
def update_config_with_globals

Reply:
Something is used when IDEA may see the usages.
Duplication != Usage

Reply:
I believe that if something is used, it is enough to say "It is used"
Where is duplication?

Reply:
This function trying to get SSL params from globals, if globals don't have SSL params, then it use defalts 
if we run update_ssl_config_with_globals("client") then we get client.jks as keystore 
if we run update_ssl_config_with_globals("server") then we get server.jks as keystore

Used in 
modules/ducktests/tests/ignitetest/services/ignite.py:65
modules/ducktests/tests/ignitetest/services/ignite_app.py:111